simpleclient_spring_boot/src/main/java/io/prometheus/client/spring/boot/PrometheusMvcEndpoint.java

```diff
@@ -0,0 +1,43 @@
+package io.prometheus.client.spring.boot;
+
+import io.prometheus.client.exporter.common.TextFormat;
+import org.springframework.boot.actuate.endpoint.mvc.EndpointMvcAdapter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.util.Set;
+
+import static org.springframework.http.HttpHeaders.CONTENT_TYPE;
+
+@ConfigurationProperties("endpoints.prometheus")
+public class PrometheusMvcEndpoint extends EndpointMvcAdapter {
```

Comment:
The impact should be minimal since users don't rely on this class explicitly; it should only be referenced by the `PrometheusEndpointConfiguration` class in the client.